testdata/resources/docker_service/updateFailsAndRollbackConvergeConfig.tf

```diff
@@ -5,12 +5,25 @@ provider "docker" {
   }
 }
 
+data "docker_registry_image" "tftest_image" {
+  provider             = "docker.private"
+  name                 = "%s"
+  insecure_skip_verify = true
+}
+resource "docker_image" "tftest_image" {
+  provider      = "docker.private"
+  name          = data.docker_registry_image.tftest_image.name
+  keep_locally  = false
+  force_remove  = true
+  pull_triggers = [data.docker_registry_image.tftest_image.sha256_digest]
```

Comment:
Does the `docker_registry_image`'s `sha256_digest` value contain `sha256:` like `RepoDigests`? -- if so, it's conceivable that at some point a different hash might be used?

Reply:
Yes it also contains the `sha256:` but without a leading container name, like `nginx` in this case
```
data_docker_image_nginx_repo_digest            = "nginx@sha256:36b74457bccb56fbf8b05f79c85569501b721d4db813b684391d63e02287c0b2"
data_docker_registry_image_nginx_sha256_digest = "sha256:36b74457bccb56fbf8b05f79c85569501b721d4db813b684391d63e02287c0b2"
```